### PR TITLE
updated welcome message

### LINKED ISSUE #6: Welcome message in the console
When the bot is started, it currently logs "I'm ready" when it is connected to Discord. 

I imagine a welcome message and maybe some information, like version number, when the bot is started and a more detailed message when the bot is connected to Discord.

### PR DESCRIPTION
**Pull request by dawksh**
Changed welcome message. This PR resolved #6. I couldn't find anything much relevant in the docs which should be added to welcome message. Added the stuff I thought was relevant

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,14 +7,15 @@ const Poker = require("./poker");
 const client = new Discord.Client();
 
 client.on("ready", () => {
-  console.log("I am ready!");
+  console.log(`Bot started on HTTP version ${client.options.http.version} on ${client.readyAt}`);
+  console.log(client)
```
Comment (Fubinator) [inline index.js]: I think you can delete the log of the whole client. It just  bloats the output

File: index.js [hunk 2]
```diff
@@ -7,14 +7,15 @@ const Poker = require("./poker");
...
 });
 
 client.on("message", (message) => {
   if (message.author.username === "planning-poker-bot") return;
 
   if (message.content === "!start") {
-    message.channel.send("Welcome to planning poker");
+    message.channel.send(`"Welcome to planning poker"`);
```
Comment (Fubinator) [inline index.js]: Could you remove the backticks here?

**Review by Fubinator — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (dawksh) [reply index.js]: oh I forgot that

Comment (Fubinator) [reply index.js]: No worries!

**Comment by DevDaksh** [comment]
maybe check the code now

**Review by Fubinator — APPROVED** [review]

**Comment by Fubinator** [comment]
Looking good @DevDaksh, I'm merging this tomorrow. Thank you for your contribution!

**Comment by dawksh** [comment]
Thanks for accepting!

**Comment by dawksh** [comment]
Would you mind adding hacktoberfest-accepted tag to this PR?

**Pull request merged**